specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/service.json

```diff
@@ -13719,7 +16109,7 @@
         },
         "lastHeartbeatUtc": {
           "format": "date-time",
-          "description": "The time of the last heartbeat received from the agent.",
+          "description": "The time of the last heartbeat recieved from the agent.",
```

Comment:
received

Reply:
done

---
In reply to: [588092528](https://github.com/AsrOneSdk/azure-rest-api-specs/pull/16#discussion_r588092528) [](ancestors = 588092528)